### DISCUSSION TITLE
change global entry padding

### DISCUSSION THREAD

**Discussion by yxdragon** [israel-dryer/ttkbootstrap#536] [Q&A] [answered]
I want to change entry's padding
```python
style = ttk.Style.get_instance()
style.configure('TEntry', padding=3)
```
but when I set the bootstyle='danger', the padding became to default back.
is there any way to change entry's padding globally? or need I config danger.TEntry and xxx.TEntry?

**Comment by israel-dryer** [comment, marked as answer]
Currently, the style is setup so that the padding it set on each style independently. This means that the padding is not inherited from the 'TEntry', and thus must be changed on the 'danger.TEntry' style directly. However, this could be fixed in a later release by setting the padding and other non-accent related options on the base TEntry allowing you to adjust that globally with a single update. I'll add this as an enhancement in a 2.x release.
